alpine/README.md

```diff
@@ -1,29 +1,30 @@
 # VividCortex Alpine Docker container
 
-Runs VividCortex agents in an [Alpine Linux](https://www.alpinelinux.org/) Docker container, for off-host monitoring (see
-[here](https://docs.vividcortex.com/getting-started/installing/#off-host-monitoring)).
+Runs VividCortex agents in an [Alpine Linux](https://www.alpinelinux.org/) Docker container, for off-host monitoring; see
+[here](https://docs.vividcortex.com/getting-started/off-host-installation/) for more detailis about off-host configuration requirements for your database. More general container installation documentation can be found [here](https://docs.vividcortex.com/getting-started/containerized-installation/).
 
 To use, you should build a image for your environment:
 
 	docker build --force-rm --build-arg VC_API_TOKEN=xxxxxxxxxxxxxxxx -t vcimage \
```

Comment:
We probably need to change this, right? For reference, the commands at the top of the Dockerfile should be updated to mirror the README

Reply:
i see, you're saying i have to change this: https://github.com/VividCortex/docker/blob/19-token/alpine/Dockerfile#L6

Reply:
Right, it should be consistent with the README